src/components/layout/layout.tsx

```diff
@@ -85,6 +87,14 @@ const Layout = ({
 
                     const current_client_country = clients_country || ''
                     const client_information_cookie = new CookieStorage('client_information')
+
+                    const new_website_status = {
+                        ...website_status,
+                        clients_country: current_client_country,
+                    }
+
+                    setWebsiteStatus(new_website_status)
```

Comment:
```suggestion
                    setWebsiteStatus((prev) => ({ ...prev, clients_country: current_client_country,}))
```
You can use the `prev` callback for the previous website_status

Reply:
Hi @yashim-deriv 

Argument of type '(prev: any) => any' is not assignable to parameter of type 'void | WebsiteStatus'.

This is the type of this
type WebsiteStatusContextType = {
    setWebsiteStatus: Dispatch<WebsiteStatus | void>
    website_status: WebsiteStatus
}